backend/src/main/java/com/sep6/backend/controller/AccountsController.java

```diff
@@ -1,18 +1,27 @@
 package com.sep6.backend.controller;
 
+import com.sep6.backend.models.Account;
 import com.sep6.backend.service.AccountsService;
+import lombok.AllArgsConstructor;
+import lombok.RequiredArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.Mapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("/accounts")
+@AllArgsConstructor
 public class AccountsController {
 
     private AccountsService service;
-    @Autowired
-    public AccountsController(AccountsService service) {
-        this.service = service;
+
+    @PostMapping(value = "{id}")
+    public void editAccount(@PathVariable int id, @RequestBody Account account){
+        //TODO: return status codes
```

Comment:
We can create exception handling according to https://spring.io/blog/2013/11/01/exception-handling-in-spring-mvc

Reply:
But this should be done in a new task

Reply:
Yep there will be separate task for this